apps/fluent-tester/src/FluentTester.tsx

```diff
@@ -135,6 +135,14 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
   const isTestSectionVisible = !enableSinglePaneView || (enableSinglePaneView && !onTestListView);
 
   const TestList: React.FunctionComponent = () => {
+    const ref = React.useRef<View>();
+
+    React.useEffect(() => {
+      if (Platform.OS === ('win32' as any)) {
+        ref.current.focus();
+      }
+    }, []);
+
```

Comment:
This would run every time TestList re-renders, rather than the whole app re-rendering. I'm not _sure_ but that may re-render more than we like. Just to confirm, clicking around different test pages doesn't cause focus to jump back to the test list?

Reply:
You're correct. Moving the block out of the testlist and into the base component fixes this.